frontend/my-app/src/components/CreateMeeting.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+
+export default function CreateMeeting() {
+    const [meetingInfo, setInfo] = useState({
+        code: "",
+        duration: "",
+        type: ""
+    })
+    const [isPending, setPending] = useState(false);
+    function submit(e) {
+        e.preventDefault();
+        setPending(true);
+        const data = {
```

Comment:
So data is supposed to be meetingInfo right? Like they look identical so can you lmk why you have both when you can just call JSON.stringify(meetingInfo)?